packages/library/src/commonMain/kotlin/io/realm/internal/RealmObjectHelper.kt

```diff
@@ -61,6 +64,16 @@ object RealmObjectHelper {
         return null
     }
 
+    fun getList(obj: RealmObjectInternal, col: String): RealmList<Any?> {
```

Comment:
Maybe add a one line comment on the type issues if you had issues making a generic variant. I know that it is listed in the class comment, but maybe also make it explicit here, so we know where we could improve if we find a way.